docs/index.org

```diff
@@ -480,9 +480,12 @@ subject of our metadata. Our metadata will be keyed by the subject, so
 when querying a metadata service for the metadata of our asset, we
 will use the subject in our query.
 
-The standard for wallet metadata is that the subject is equal to the
-hash of the policy concatenated to the base16-encoded asset name
-(which can be empty).
+For phase-1 monetary scripts the standard for wallet metadata is that
+the subject is equal to the hash of the policy concatenated to the
+base16-encoded asset name (which can be empty).
+
+For phase-2 Plutus Smart-Contracts the ~POLICY_ID~ is equal to the
```

Comment:
Do you mean?
```suggestion
For phase-2 Plutus Smart-Contracts the subject is equal to the
```

I think also it would be very useful to provide a command how to obtain it as it is provided for subject for tokens below.